apps/blockchain/lib/blockchain/ethash/cache.ex

```diff
@@ -0,0 +1,4106 @@
+defmodule Ethash.Cache do
+  @data_sizes [
+    1_073_739_904,
```

Comment:
maybe we can move these values to text files?

Reply:
Agree, this can be loaded from a file in app start-up

Reply:
Sure, this means they would be baked into the compiled code? I think that's a fine solution.